common/app/views/support/OmnitureAnalyticsData.scala

```diff
@@ -2,12 +2,25 @@ package views.support
 
 import java.net.URLEncoder._
 
+import conf.Configuration
 import model.{Content, MetaData}
 import play.api.libs.json.{Json, JsValue, JsString}
 import play.api.mvc.RequestHeader
 import play.twirl.api.Html
 
+object OmnitureAnalyticsAccount {
+  def apply(page: MetaData): String = {
+    val sectionSpecficAccounts = Map(
+      ("guardian-masterclasses", "guardiangu-masterclasses"),
+      ("Guardian Masterclasses", "guardiangu-masterclasses")
+    )
+    val account = Seq(Some(Configuration.omniture.account), sectionSpecficAccounts.get(page.section)).flatten.mkString(",")
+    account
```

Comment:
Given you're just immediately returning the val, might as well not have the val.